src/main.rs

```diff
@@ -52,13 +55,15 @@ init_i18n!(
     "plume", ar, bg, ca, cs, de, en, eo, es, fr, gl, hi, hr, it, ja, nb, pl, pt, ro, ru, sr, sk, sv
 );
 
+mod api;
 mod mail;
 #[cfg(feature = "test")]
 mod test_routes;
+mod web;
 
 compile_i18n!();
 
-fn main() {
+fn main() -> std::io::Result<()> {
```

Comment:
What's this do?

Reply:
It allows to use `?` in `main` basically. And if we return an error the program will exit with something else than `0`.